SPECS/cloud-init/cloud-init.spec

```diff
@@ -74,8 +74,6 @@ Cloud-init configuration for Hyper-V telemetry
 %prep
 %autosetup -p1 -n %{name}-%{version}
 
-find systemd -name "cloud*.service*" | xargs sed -i s/StandardOutput=journal+console/StandardOutput=journal/g
```

Comment:
Can you update the PR description with why this change fixes the issue with the serial console?

i.e., https://www.freedesktop.org/software/systemd/man/systemd.exec.html#StandardOutput=